app/controllers/buyers/service_contracts_controller.rb

```diff
@@ -16,8 +16,8 @@ def index
     @search = ThreeScale::Search.new(params[:search] || params)
     @plans = current_account.service_plans
 
-    if params[:service_id]
-      @service = @services.find params[:service_id]
+    if (service_id = params[:service_id] || @search.service_id)
```

Comment:
Not sure if we should land search in service context. Do we do this with applications?

Reply:
@thomasmaas 
I don't know how we do it for `applications` but it is not necessarily the same... the `under service` link you told me about, it actually sends the param through `search` 😄 and if there are only those 3 you told me, it may mean that it is always through `search` and there is not a real search page anywhere 🤔

Reply:
yes, later I saw that and merged…